### DISCUSSION TITLE
Feature Request - Age Restriction Bypass

### DISCUSSION THREAD

**Discussion by ghost** [TeamVanced/VancedManager#631] [Ideas]
There's a useful script called "Simple YouTube Age Restriction Bypass" that allows you to bypass age restriction when using YouTube on a web browser. Is it possible to implement a feature like this into YouTube Vanced the same way you've implemented Sponsorblock into YouTube Vanced?

**Comment by X1nto** [comment]
This is not meant for discussing Vanced features
